web/packages/teleport/src/Login/Login.test.tsx

```diff
@@ -19,15 +19,21 @@ import { render, fireEvent, screen, waitFor } from 'design/utils/testing';
 import { privateKeyEnablingPolicies } from 'shared/services/consts';
 
 import auth from 'teleport/services/auth/auth';
-import history from 'teleport/services/history';
 import cfg from 'teleport/config';
 
 import Login from './Login';
 
+const assignMock = jest.fn();
+
+delete window.location;
```

Comment:
This seems like it could potentially cause cascading test failures. Can you explicitly reset this value plz.